.github/workflows/deploy.yml

```diff
@@ -4,6 +4,12 @@ on:
   push:
     branches:
       - master
+    paths:
+    - '!.github/**'
```

Comment:
If we make a change to the workflow, we want to run it I guess, so this shouldn't be excluded?

Reply:
My thinking was that as the workflow YML file itself is not part of the eventual app, then the next push which has a change to the app would be deployed using the new workflow. Possibly this decouples the feedback loop, so I'm happy to remove it.

Reply:
Yes I saw that issue in my previous job and it was really annoying.

Reply:
Removed!